sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.students.controller.getApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  FakeStudentUsersTypes,
+  getStudentToken,
+  InstitutionTokenTypes,
+  getAuthRelatedEntities,
+  getStudentByFakeStudentUserType,
+} from "../../../../testHelpers";
+import {
+  saveFakeApplicationOfferingRequestChange,
+  createE2EDataSources,
+  E2EDataSources,
+  createFakeInstitutionLocation,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { InstitutionLocation, Student } from "@sims/sims-db";
+
+describe("ApplicationOfferingChangeRequestStudentsController(e2e)-getApplicationOfferingChangeRequest", () => {
+  let app: INestApplication;
+  let collegeFLocation: InstitutionLocation;
+  let student: Student;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation({ institution: collegeF });
```

Comment:
do we need an existing location, I belive a dummy location should also be fine right?